docker-compose.dcproj

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What is this used for?

Reply:
Can you please expand on what docker prj is?

Reply:
I will check with Huss on this.